ckanext/canada/templates/internal/admin/publish_search.html

```diff
@@ -55,7 +55,11 @@ <h2 class="mrgn-tp-0 mrgn-bttm-0">
           {% block filter_list %}
             {% for field in g.fields_grouped %}
               {% if field != 'ready_to_publish' and field != 'imso_approval' %}
-                {{ h.search_filter_pill(field, extras) }}
+                {% for link_label in h.search_filter_pill_link_label(field, extras) %}
+                  <a href="{{ link_label[0] }}" class="btn btn-info btn-xs mrgn-lft-sm mrgn-bttm-sm facet-pill" title="{{ _('Remove') }}">
+                    {{ link_label[1] }}&nbsp;<i class="fa fa-times" aria-hidden="true"></i></a>
```

Comment:
this is prettier and loudly fails if you get anything but two items back
```suggestion
                {% for link, label in h.search_filter_pill_link_label(field, extras) %}
                  <a href="{{ link }}" class="btn btn-info btn-xs mrgn-lft-sm mrgn-bttm-sm facet-pill" title="{{ _('Remove') }}">
                    {{ label }}&nbsp;<i class="fa fa-times" aria-hidden="true"></i></a>
```